code_samples/simple_language_plugin/src/main/gen/org/intellij/sdk/language/SimpleLexer.java

```diff
@@ -11,7 +11,7 @@
 
 /**
  * This class is a scanner generated by 
- * <a href="http://www.jflex.de/">JFlex</a> 1.7.0
+ * <a href="https://www.jflex.de/">JFlex</a> 1.7.0
```

Comment:
Please don't edit generated files. This looks like a link embedded in a JavaDoc comment written by the jflex tool.